packages/store/addon/-private/system/store.ts

```diff
@@ -1180,7 +1289,10 @@ class Store extends Service {
     @param options optional to include adapterOptions
     @return {Promise} promise
   */
-  _reloadRecord(internalModel, options) {
+  _reloadRecord(internalModel, options): RSVP.Promise<InternalModel> {
+    if (REQUEST_SERVICE) {
+      options.isReloading = true;
```

Comment:
I prefer `isReloading` but consider that elsewhere and in public API we use `reload` as this flag. Could cause confusion to users.

Reply:
since it is not an adapterOption we may not leak this in which case this point does not matter :)